examples/01_encodings.py

```diff
@@ -0,0 +1,195 @@
+
+from pprint import pprint
+
+# Recover the |TableVectorizer| from the |Pipeline|
+tv = pipeline.named_steps["tablevectorizer"]
+
+pprint(tv.transformers_)
+
+###############################################################################
+# We observe it has automatically assigned an appropriate encoder to
+# corresponding columns:
+
+###############################################################################
+#     - The |OneHotEncoder| for low cardinality string variables, the columns
+#       'gender', 'department', 'department_name' and 'assignment_category'.
+
+feature_names = tv.get_feature_names_out()
+
+feature_names[4:82]
+
+###############################################################################
+#     - The |GapEncoder| for high cardinality string columns, 'employee_position_title'
+#       and 'division'. The |GapEncoder| is a powerful encoder that can handle dirty
+#       categorical columns.
+
+feature_names[82:]
+
+###############################################################################
+#     - The |DatetimeEncoder| to the 'date_first_hired' column. The |DatetimeEncoder|
+#       can encode datetime columns for machine learning.
+
+feature_names[0:4]
+
+###############################################################################
+# As we can see, it gave us interpretable column names.
+#
+# In total, we have a reasonable number of encoded columns:
+
+len(feature_names)
+
+###############################################################################
+# Let's look at the cross-validated R2 score of our model:
+
+from sklearn.model_selection import cross_val_score
+import numpy as np
+
+scores = cross_val_score(pipeline, X, y)
+print(f"R2 score:  mean: {np.mean(scores):.3f}; std: {np.std(scores):.3f}\n")
+
+###############################################################################
+# The simple pipeline applied on this complex dataset gave us very good results.
+
+###############################################################################
+# Feature importances in the statistical model
+# --------------------------------------------
+#
+# In this section, after training a regressor, we will plot the feature importances.
+#
+# .. topic:: Note:
+#
+#   To minimize computation time, we use the feature importances computed by the
+#   |RandomForestRegressor|, but you should prefer |permutation importances|
+#   instead (which are less subject to biases).
+#
+# First, let's train another scikit-learn regressor, the |RandomForestRegressor|:
+
+from sklearn.ensemble import RandomForestRegressor
+
+regressor = RandomForestRegressor()
+
+pipeline = make_pipeline(TableVectorizer(), regressor)
+pipeline.fit(X, y)
+
+###############################################################################
+# We are retrieving the feature importances:
+
+importances = regressor.feature_importances_
+std = np.std([tree.feature_importances_ for tree in regressor.estimators_], axis=0)
```

Comment:
Nitpick: improve readability
```suggestion
avg_importances = regressor.feature_importances_
std_importances = np.std([tree.feature_importances_ for tree in regressor.estimators_], axis=0)
```